src/input.service.ts

```diff
@@ -278,21 +336,23 @@ export class InputService {
   private subscriptions: Subscription[] = [];
   private pollRaf: number = null;
 
-  constructor(private focus: FocusService) {}
+  constructor(private focus: FocusService) { }
 
   /**
    * Bootstrap attaches event listeners from the service to the DOM and sets
    * up the focuser rooted in the target element.
    */
-  public bootstrap(root: HTMLElement = document.body) {
+  public bootstrap(handleTab : (direction : Direction) => any = (direction : Direction) => { direction }, root: HTMLElement = document.body) {
+    this.handleTab = handleTab;
```

Comment:
It could, but I would probably have separate public `EventEmitters` which fire (from `@angular/core`) on the focus service rather than providing a callback in the bootstrap